### DISCUSSION TITLE
Replacement for centos7

### DISCUSSION THREAD

**Discussion by Yuvaraj2930** [coder/code-server#6605] [Q&A]
hi we are currently running centos7 in our multiple for our Servers as the support of centos7 is going to end we an clear idea of chnaging our os in our production line can you kindly help us out on it we have os kindly suggest which we can move on

Alma Linux 8 - EOL 01-Mar-2029

Rocky Linux 8 - EOL 31-May-2029

Centos Stream 9 - EOL 31-May-2027

RHEL 9 - EOL 31-May-2032

and most of the ppl are not toking about rhel its has an subscribing but they issuse free pathces and security till eol can you guys help me out in this

**Comment by code-asher** [comment]
Is this about code-server specifically or just generally asking about migrating from CentOS 7?  If the latter I would recommend checking elsewhere, this forum is just for code-server issues so we are not able to provide any support beyond that.
